oktoberfest/ce_calibration.py

```diff
@@ -149,11 +161,11 @@ def gen_lib(self, df_search: Optional[pd.DataFrame] = None):
 
     def get_hdf5_path(self) -> str:
         """Get path to hdf5 file."""
-        return self.out_path + ".hdf5"
+        return str(self.out_path) + ".hdf5"
 
     def get_pred_path(self) -> str:
         """Get path to prediction hdf5 file."""
-        return self.out_path + "_pred.hdf5"
+        return str(self.out_path) + "_pred.hdf5"
```

Comment:
same here... Maybe we can also use ".pred.hdf5", since that would allow the usage of with_suffix method. Can you check if that is a breaking change further downstream? I.e. changing undersocre to dot?